typescript/ecs/fargate-service-with-local-image/index.ts

```diff
@@ -18,9 +17,7 @@ const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
 // with the image from ECR.
 new ecs_patterns.LoadBalancedFargateService(stack, "FargateService", {
   cluster,
-  image: ecs.ContainerImage.fromAsset(stack, "local-image" , { 
-    directory: path.join(__dirname, 'local-image') 
-  })
+  image: ecs.ContainerImage.fromAsset("local-image")
```

Comment:
You want to pass in the directory here `path.resolve(__dirname, 'local-image')` so that image looks like this:
`image: ecs.ContainerImage.fromAsset("local-image")`

Reply:
a39b73cef38cb4b1c20d4432c9a9852ce8431279 addresses this issue